wandb/integration/sb3/__init__.py

```diff
@@ -0,0 +1,111 @@
+"""W&B callback for sb3
+
+Really simple callback to get logging for each tree
+
+Example usage:
+
+```python
+import gym
+import time
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import DummyVecEnv
+from wandb.integration.sb3 import WandbCallback
+import wandb
+
+# initialize the experiment
+# note that you have to call `wandb.init(...)` before using WandbCallback
+config = {
+    "policy_type": "MlpPolicy",
+    "total_timesteps": 25000
+}
+experiment_name = f"PPO_{int(time.time())}"
+wandb.init(
+    name=experiment_name,
+    project="sb3",
+    config=config,
+    allow_val_change=True, # auto-populate config with the model's hyperparams
+    sync_tensorboard=True, # auto-upload metrics
+    monitor_gym=True,  # auto-upload the videos of agents playing the game
+    save_code=True,  # optional
+)
+
+# initialize the run
+def make_env():
+    env = gym.make("CartPole-v1")
+    env = gym.wrappers.Monitor(env, f"videos/{experiment_name}", force=True) # record videos
+    env = gym.wrappers.RecordEpisodeStatistics(env) # record stats such as returns
```

Comment:
I know that would be confusing but I guess it will work too with a SB3 `Monitor` instead of the `RecordEpisodeStatistics` wrapper?

if so, best is to probably add a comment no?
same question when using `VecMonitor`

Reply:
Maybe I should just use `stable_baselines3.common.monitor.Monitor(env)` instead of the `RecordEpisodeStatistics` to avoid confusion. I don't think `RecordEpisodeStatistics` is widely used. What do you think?

Reply:
yes but add a comment then to avoid confusion, because we will have two "Monitor" wrappers doing two different things, no?

Reply:
Good point! You know what I will just stick with SB3 completely and use the `VideoRecorder` instead